docs/notebooks/spatio_temporal.py

```diff
@@ -36,35 +37,37 @@
 # -
 
 # Creating data
-num_data = 200
-time_points = np.random.rand(num_data, 1)
+num_data = 500
+std_noise = 0.5
+time_points = np.random.uniform(0, T, num_data).reshape(-1, 1)
 space_points = np.random.rand(num_data, 1)
 X = np.concatenate([space_points, time_points], -1)
 f = lambda v: np.cos(5.0 * (v[..., 1:] + v[..., :1]))
 F = f(X)
-Y = F + np.random.randn(num_data, 1)
+Y = F + np.random.randn(num_data, 1) * std_noise
 data = (X, Y)
 
 # Creating a plotting grid and plotting function
 
 # +
-x_grid, t_grid = np.meshgrid(np.linspace(0, 1, 50), np.linspace(0, 1, 50))
+x_grid, t_grid = np.meshgrid(np.linspace(0, 1, 50), np.linspace(0, T, 50))
 X_grid = np.concatenate([x_grid.reshape(-1, 1), t_grid.reshape(-1, 1)], axis=-1)
 
 
 def plot_model(model):
     mu_f, var_f = model.space_time_predict_f(X_grid)
     fig, axarr = plt.subplots(2, 1)
-    axarr[0].scatter(x=space_points, y=time_points, c=Y)
-    axarr[1].scatter(x=X_grid[..., :1], y=X_grid[..., 1:], c=mu_f.numpy())
+    axarr[0].scatter(x=time_points, y=space_points, c=Y)
+    axarr[1].scatter(x=X_grid[..., 1:], y=X_grid[..., :1], c=mu_f.numpy())
 
-    for ax in axarr:
-        ax.hlines(model.inducing_space, xmin=time_points.min(), xmax=time_points.max(), colors="r")
-        ax.vlines(
-            model.inducing_space, ymin=space_points.min(), ymax=space_points.max(), colors="k"
-        )
+    axarr[1].hlines(
+        model._inducing_space, xmin=time_points.min(), xmax=time_points.max(), colors="r"
+    )
+    axarr[1].vlines(
+        model._inducing_time, ymin=space_points.min(), ymax=space_points.max(), colors="k"
+    )
```

Comment:
Since the model defines `inducing_space` and `inducing_time` property methods, couldn't you use those?
```suggestion
    axarr[1].hlines(
        model.inducing_space, xmin=time_points.min(), xmax=time_points.max(), colors="r"
    )
    axarr[1].vlines(
        model.inducing_time, ymin=space_points.min(), ymax=space_points.max(), colors="k"
    )
```